glossary.ftd

```diff
@@ -1,22 +1,219 @@
 -- ft.page: Glossary
 
-For readers/writers of document by id, we have three kinds of id: `id: /foo/`,
-`full-id: /foo/-/x/`, `filename: foo.ftd`.
+An FPM Glossary
 
 
-Since we need to get `id`, `full-id` and `filename` of current document, to power
-document-readers, lets create processors `document-id`, `document-full-id` and
-`document-filename` as helpers as well. These three processors will return ids
-for current document.
+
+-- ft.h2: client project
+
+An [fpm project](glossary/#fpm-project) which is a clone to the
+[server project](glossary/#server-project) and tracks it. The interaction
+with server project is established using [sync](glossary/#sync).
+
+
+-- ft.h2: clone
+
+Clone a [server project](glossary/#server-project) into a new directory
+which becomes a [client](glossary/#server-project).
+
+
+-- ft.h2: conflict-state
+
+The file is said to be in the `conflict-state` if the file gets in conflict with
+server latest [version](glossary/#version) of the same file.
+
+```
+The file, in conflict-state, is never considered to be synced unless its
+conflict is resolved.
+```
+
+
+-- ft.h2: document
```

Comment:
We should mention that `module` somewhere. Like a document is a module. When we are importing, we are importing a package or a module.

Reply:
I think module is a what you are describing here. We should leave `document` to mean any file, not just a valid "module". To me module is what you can import. Technically you can not even import markdown. But markdown and ftd documents you can render, so we need a term to describe both markdown and ftd file, which is what you mean here.